### PR TITLE
ARTEMIS-4617 upgrade JLine to 3.25.1

### PR DESCRIPTION
**Pull request by clebertsuconic**
I had to remove the indirect dependency between the maven plugin and jline to avoid the maven plugin to parse some classes in jline that require experimental features on the JDK even when they are not in use.

### CODE REVIEW

**Comment by jbertram** [comment]
According to [this](https://github.com/jline/jline3?tab=readme-ov-file#jansi) Jansi should be integrated into 3.25.x so we should be able to get ride of the Jansi dependency now.

**Comment by clebertsuconic** [comment]
@jbertram ohh.. jansi.. I think I reverted one change by accident... I will try to remove it now

File: artemis-distribution/pom.xml [hunk 1]
```diff
@@ -157,6 +157,16 @@
          <version>${project.version}</version>
       </dependency>
 
+      <!-- shell integration -->
```
Comment (clebertsuconic) [inline artemis-distribution/pom.xml:160]: I don't need this any more.

**Comment by clebertsuconic** [comment]
this was previously discussed here: https://github.com/apache/activemq-artemis/pull/4790

this PR superseded #4790

**Pull request merged**